vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/bean/boutiquesTools/BoutiquesApplication.java

```diff
@@ -266,6 +302,38 @@ public void setApplicationLFN(String applicationLFN) {
         this.applicationLFN = applicationLFN;
     }
 
+    public void setCustomProperties(Set<String> inputIds) {
+        this.customProperties = inputIds;
+    }
+
+    public static void setCommandLineFlag(Set<String> commandLineFlag) {
+        BoutiquesApplication.commandLineFlag = commandLineFlag;
+    }
+
+    public String setWarning() {
+        Set<String> commonValues = new HashSet<>();
+        for (String customValue : customProperties) {
+            // If the value is found in commandLineFlags set, add it to commonValues set
+            if (commandLineFlag.contains(customValue)) {
+                commonValues.add(customValue);
+            }
+        }
+        if (commonValues == null || commonValues.isEmpty()) {
+            return "";
+          }
+          // Use a StringBuilder to efficiently build the string
+          StringBuilder formattedString = new StringBuilder();
+          boolean first = true;
+          for (String property : commandLineFlag) {
+            if (!first) {
+              formattedString.append(", ");
+            }
+            formattedString.append(property);
+            first = false;
+          }
+        return formattedString.toString();
+    }
+
```

Comment:
This must be done of the client side where there are already boutiques verifications.

Reply:
moved setWarning intelligence to boutiques parser